### PR TITLE
feat(Pipeline migration to 1ES template): Migrated Accessibility Insights Web CI and Accessibility Insights Web Manual to 1ES PT

### PR DESCRIPTION
**Pull request by v-prachin**
#### Details

Migrated Accessibility Insights Web CI and Accessibility Insights Web Manual to 1ES PT as per the changes mentioned in the documentation link provided for the 1ES pipeline migration.

We have three pipelines:
1.accessibility-insights-web CI: Non-Production tag ---> accessibility-insights 
2. Accessibility Insights Web Manual: Non-Production tag ---> accessibility-insights-private 
3. Accessibility Insights Web CI: Production tag ---> accessibility-insights-private

**Accessibility Insights Web CI:**  

1.Added a new file build-webCI.yaml with Official template in it as this pipeline has a production tag.
2.Created a new pipeline microsoft.accessibility-insights-web pointing it to build-webCI.yaml for testing purpose.
3.The pipeline is successfully validated and able to run it successfully.
https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=49589&view=results

**accessibility-insights-web CI:** 

1.As this pipeline has Non-Production tag and it is situated in accessibility-insights project in DevOps. This pipeline is originally pointing to build.yaml, so made the template their Unofficial & made other changes necessary to migrate the pipeline to 1ES.
2.Validation & run successful 
https://dev.azure.com/accessibility-insights/accessibility-insights-web/_build/results?buildId=50974&view=results
 
**Accessibility Insights Web Manual:** 

1.As this pipeline has Non-Production tag and it is situated in accessibility-insights-private project in DevOps. This pipeline is originally pointing to build.yaml, so made the template their Unofficial & made other changes necessary to migrate the pipeline to 1ES.
2.Validation successful and the run is also successful. However, there is a scenario where the pipeline fails at step linux run docker entrypoint. After multiple re-runs it succeeds.
So here we are having bit of an issue where the pipeline sometimes runs successfully and sometimes not. Even though the change made is exactly similar to the other two pipelines and they are successful there.
Successful attempt: https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=49538&view=results
Failed attempt: https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=49583&view=results

NOTE: microsoft.accessibility-insights-web(1) was a pipeline created for testing purpose and then it is deleted as it was no longer required. Thus that check is failing, as this pipeline no longer exists.

##### Motivation

Pipeline migration to 1ES template. User stories -
[User Story 2126537](https://dev.azure.com/mseng/1ES/_workitems/edit/2126537) , [User Story 2126529](https://dev.azure.com/mseng/1ES/_workitems/edit/2126529) and [User Story 2126531](https://dev.azure.com/mseng/1ES/_workitems/edit/2126531)

##### Context
** For Accessibility Insights Web CI:**   
Once this PR change is approved, the plan is to point the existing pipeline "Accessibility Insights Web CI" (with Production tag) to build-webCI.yaml instead of build.yaml.
Also delete the pipeline created for testing purpose i.e. microsoft.accessibility-insights-web

#### Pull request checklist

- [x] Addresses an existing issue: [User Story 2126537](https://dev.azure.com/mseng/1ES/_workitems/edit/2126537) , [User Story 2126529](https://dev.azure.com/mseng/1ES/_workitems/edit/2126529) and [User Story 2126531](https://dev.azure.com/mseng/1ES/_workitems/edit/2126531)
- [x] Ran `yarn fastpass`
- [n/a ] Added/updated relevant unit test(s) (and ran `yarn test`)
- [n/a ] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [n/a] (UI changes only) Added screenshots/GIFs to description above
- [n/a ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Comment by v-viyada** [comment]
the check failing for pipeline "microsoft.accessibility-insights-web (1) " is expected. This was a test pipeline which got deleted after testing.

File: build-webCI.yaml [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+extends:
+    template: v1/1ES.Official.PipelineTemplate.yml@1esPipelines
+    parameters:
+        pool:
+            name: $(a11yInsightsPool) # Name of your hosted pool
+            image: windows-2022-secure # Name of the image in your pool. If not specified, first image of the pool is used
+            os: windows
+
+        stages:
```
Comment (DaveTryon) [inline build-webCI.yaml:35]: @v-prachin it looks like lines 35-182 of this file are identical to lines 41-188 of build.yaml. That's an 80% overlap in terms of lines., which almost guarantees that they'll eventually get out of sync.  Is there a way that we can consolidate these 147 lines into a single yml file for maintenance purposes?

File: pipeline/e2e-test-publish-results.yaml [hunk 2]
```diff
@@ -8,10 +9,3 @@ steps:
       condition: succeededOrFailed()
       displayName: publish e2e test results
       timeoutInMinutes: 3
-
-    - publish: '$(System.DefaultWorkingDirectory)/test-results/e2e'
```
Comment (DaveTryon) [inline pipeline/e2e-test-publish-results.yaml:12]: This gets pulled in from `e2e-job-per-environment.yaml`. Have we considered the impact of removing these lines in that context?

File: build-webCI.yaml (hunk 1 above)
Comment (v-prachin) [reply build-webCI.yaml]: Added a new yml accessibility-insights-web.yml for this suggestion.

File: pipeline/e2e-test-publish-results.yaml (hunk 2 above)
Comment (v-prachin) [reply pipeline/e2e-test-publish-results.yaml:12]: Initially removed this from here and directly included it in the job itself as template Context. However, now I have included this part as a task in this file itself.

File: pipeline/accessibility-insights-web.yml [hunk 3]
```diff
@@ -0,0 +1,166 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (DaveTryon) [inline pipeline/accessibility-insights-web.yml:1]: The file contents look fine, but I wonder if we can give this file a more descriptive name. Maybe `build-shared.yaml` or `build-common.yaml` ?

**Review by DaveTryon — APPROVED** [review]
@v-prachin I made a suggestion about renaming the file with the shared yaml content. No need to re-review once the file has been renamed.

Nice work!

File: pipeline/accessibility-insights-web.yml (hunk 3 above)
Comment (v-prachin) [reply pipeline/accessibility-insights-web.yml]: Changed the file name to build-shared.yaml

**Pull request merged**